src/main/java/page_objects/GeneralPage.java

```diff
@@ -11,6 +11,7 @@
     private final Tab logOutTab = new Tab(By.cssSelector("a[href='/Account/Logout']"));
     private final Tab changePwdTab = new Tab(By.cssSelector("a[href='/Account/ChangePassword.cshtml']"));
     private final Tab MyTicketTab = new Tab(By.cssSelector("a[href='/Page/ManageTicket.cshtml']"));
+    private final Tab ContactTab = new Tab(By.cssSelector("a[href='/Page/Contact.cshtml']"));
```

Comment:
naming convention

Reply:
this is unacceptable

Reply:
I told you about this at the very beginning of class and now you make this mistake